test/e2e/apimachinery/resource_quota.go

```diff
@@ -1478,7 +1478,10 @@ func newTestResourceQuota(name string) *v1.ResourceQuota {
 	hard[v1.ResourceQuotas] = resource.MustParse("1")
 	hard[v1.ResourceCPU] = resource.MustParse("1")
 	hard[v1.ResourceMemory] = resource.MustParse("500Mi")
-	hard[v1.ResourceConfigMaps] = resource.MustParse("2")
+	// Update configmap quota from 2 to 3 to account for the addition of
```

Comment:
Double-entry commenting:

This change is required for 4.7 to ensure the test can pass with the addition of the new service ca configmap. The test code was updated in 1.21 (4.8 and above) to set the configmap quota to 10, so this change was not required for the addition of service ca configmap publishing to pass conformance in the 4.8 PR.